src/state/slices/opportunitiesSlice/constants.ts

```diff
@@ -100,13 +100,13 @@ export const STAKING_ID_DELIMITER = '*'
 export const DEFI_PROVIDER_TO_METADATA: Record<DefiProvider, DefiProviderMetadata> = {
   [DefiProvider.ShapeShift]: {
     provider: DefiProvider.ShapeShift,
-    icon: 'https://assets.coincap.io/assets/icons/256/fox.png',
+    icon: 'https://assets.coingecko.com/coins/images/9988/standard/FOX.png?1696510025',
     color: '#3761F9',
     url: 'https://app.shapeshift.com',
   },
   [DefiProvider.EthFoxStaking]: {
     provider: DefiProvider.EthFoxStaking,
-    icon: 'https://assets.coincap.io/assets/icons/256/fox.png',
+    icon: 'https://assets.coingecko.com/coins/images/9988/standard/FOX.png?1696510025',
```

Comment:
```suggestion
    icon: 'https://coin-images.coingecko.com/coins/images/9988/large/FOX.png?1696510025',
```

Reply:
Fixed in [d50ae4f](https://github.com/shapeshift/web/pull/7298/commits/d50ae4fbe0b84892069e74dfc28a17a7599c9e16)